src/charts/createTaskChart.js

```diff
@@ -0,0 +1,55 @@
+const d3 = require('d3');
+
+/**
+Percentage is a decimal integer.  
+
+Function creates a donut chart based on percentage and color to differentiate
+between Complete, In Progress, and Todo.
+**/
+
+export function createTaskChart(percentage, color) {
```

Comment:
The name of this function implies it is creating a chart based on a data point (percentage) and color. And it appears that is what is going on in that a node is being built up and returned.
However at the point where this function is called it is captured in variables named data and then sent as props to to another component. Overall I find this usage a bit unintuitive.
It could just be the nature of the library being used though so perhaps just fine.